src/ipc/Queue.h

```diff
@@ -410,6 +426,96 @@ OneToOneUniQueue::push(const Value &value, QueueReader *const reader)
     return wasEmpty && (!reader || reader->raiseSignal());
 }
 
+template <class Value>
+void
+OneToOneUniQueue::statIn(StoreEntry &entry) const
+{
+    // Nobody can modify our theIn so, after capturing some valid theSize value
+    // in count, we can reliably report all [theIn-count, theIn) items that were
+    // queued at theSize capturing time. We may report items already pop()ed by
+    // the other side, but that is OK because pop() does not modify items -- it
+    // only increments theOut.
+    const auto count = theSize.load();
+    statOpen(entry, "pushIndex", "other", count);
+    statClose<Value>(entry, theIn - count, count); // Branch XXX: theIn-count may get negative!
```

Comment:
I forgot to remove this XXX:

```suggestion
    statClose<Value>(entry, theIn - count, count); // unsigned underflow is OK
```

Reply:
Fixed at 54470d8.